extraConfigMicroshift.py

```diff
@@ -12,7 +12,7 @@ def wait_for_microshift(acc: host.Host, kubeconfig: str) -> None:
     for attempt in range(1, 21):
         ret = acc.run(f"oc get no --kubeconfig {kubeconfig}")
         if ret.returncode == 0:
-            if "Ready" in ret.out:
+            if "Ready" in ret.out and "NotReady" not in ret.out:
```

Comment:
A little awkward... oc get nodes -ojson?

Reply:
Yeah good point, reworked this check to use jsonpath instead